bee-tangle/src/testhelpers.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 IOTA Stiftung
```

Comment:
Following the discussion with Joshua, I am now even more convinced this should be in the `tests` folder :)

Reply:
let me think about it ;)

Reply:
I definitely need this test setup for my unit tests. By the way, it is pretty common to have test setup (and teardown) next to your unit tests. Putting it in a separate file might be something that is not necessary very often, so I can offer you to move it ;)

Reply:
I think I can put it in `bee_test`? Would you be happy with that?

Reply:
So just trying to understand because maybe I'm missing something: are they really unit tests in the sense we discussed ? I feel like traversals are a public API so they should be tested as integration tests in `tests` ? But maybe I overlooked some visibility

Reply:
yea, I am using the test tangle not only for testing the traversals. That's the whole reason why I created a separate file for it. To bring a concrete example: I use it also to just test the `num_tips`  function. There will be some more unit tests requiring it. Basically any Tangle method, that can only be well tested with a pre-configured Tangle.

Reply:
Alright, your crate your call :)
But just as a final comment, `num_tips` is also public so I'm not entirely sure why it would be an example against what I was proposing.